desafio-02/AnaPaulaMazi/javascript/#2Primos.js

```diff
@@ -0,0 +1,20 @@
+function verificaPrimo(num){
+let primo = true;
+if(num<=1){ primo = false};
```

Comment:
Talvez mover a linha 3 pra cima, pois voce não precisa inicializar nada se o numero for < 3 (o primeiro primo). E nesse caso voce pode retornar direto.

Tipo
```
if (num < 3) {
  return false;
}
let primo = true;
```
(continue o seu código aqui)

Reply:
Eu entendi.....mais  o primeiro numero primo não seria o 2 ? daí (num <2).... mudando a linha de código para cima..como você está dizendo. Seria isso?

Reply:
Correto. Onde eu escrevi `< 3` deveria ser '< 2`. Eu vi que voce já modificou.